### DISCUSSION TITLE
Demande d'assistance pour mon compte signalé

### DISCUSSION THREAD

**Discussion by jean-chr** [community/community#114469] [New to GitHub]
Bonjour,
Je vous contacte car mon compte a été récemment signalé sur votre plateforme, et j'aimerais obtenir des informations supplémentaires sur la raison de ce signalement. Je suis surpris par cette situation car je suis un utilisateur respectueux des règles de la communauté et je n'ai pas connaissance d'avoir enfreint les conditions d'utilisation.

Pouvez-vous s'il vous plaît me fournir des détails sur les raisons du signalement, ainsi que des informations sur la procédure à suivre pour résoudre ce problème ? Je suis prêt à coopérer pleinement pour rectifier toute erreur ou malentendu.

Merci d'avance pour votre aide et votre compréhension.

Cordialement,

**Comment by benogoulart** [comment]
Bonjour,

Si votre compte a été signalé ou soumis à des restrictions, seul le **support GitHub** peut consulter les raisons exactes. Les membres de la communauté n'ont pas accès aux informations concernant les comptes ni aux détails des signalements.

Si vous pensez qu'il s'agit d'une erreur, je vous recommande de :

- Vérifier si vous avez reçu un e-mail de GitHub expliquant les restrictions appliquées à votre compte.
- Répondre à votre ticket d'assistance existant (ou en ouvrir un si ce n'est pas déjà fait) plutôt que d'en créer plusieurs.
- Patienter pendant que le support examine votre dossier, car seul GitHub Support peut lever une restriction ou fournir des informations supplémentaires.

J'espère que votre situation sera résolue rapidement.

**Comment by v-coder247** [comment]
Hi @jean-chr, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
